src/aiperf/dataset/loader/mooncake_trace.py

```
@@ -54,6 +53,25 @@ def __init__(
         self._end_offset = user_config.input.fixed_schedule_end_offset
         super().__init__(user_config=user_config, **kwargs)
 
+    @classmethod
+    def can_load(
+        cls, data: dict[str, Any] | None = None, filename: str | Path | None = None
```

Ruff (0.14.1)

Unused class method argument: `filename`

(ARG003)